.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
```

Comment:
This is why it doesn't work (should be renamed main)

Reply:
BTW same issue on the following workflows :
- pr.yml
- benchmark.yml
- codeql-analysis.yml
- protobuf.yml
- go.yml